test/views/helpers/index.js

```diff
@@ -0,0 +1,36 @@
+var assert = require('assert');
+var helpers = require('../../../views/helpers');
+
+
+describe('views.helpers', () => {
+  describe('#contrastColor', () => {
+    it('should return "white" when the value is not present', () => {
```

Comment:
nit: don't use 'should' it just adds noise, instead: 'returns "white" if value not present'